CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+### Added
+- PRE and POST interaction commands
```

Comment:
Write here something like "this enables to lower the volume of shairport-sync when talking to Alexa"